sopel/loader.py

```diff
@@ -67,7 +68,7 @@ def clean_callable(func, config):
             func.rule = [func.rule]
         func.rule = [compile_rule(nick, rule, alias_nicks) for rule in func.rule]
 
-    if hasattr(func, 'commands') or hasattr(func, 'nickname_commands'):
+    if hasattr(func, 'commands') or hasattr(func, 'nickname_commands') or hasattr(func, 'action_commands'):
```

Comment:
I know it isn't that much shorter, but it might help line length here to do

`if all(hasattr(func, attr) for attr in ['commands', 'nickname_commands', 'action_commands']):`

Reply:
Using `all()` would break it.

Reply:
Yes, `any` would be cool.